src/hooks/useGlobalEndCallKey.js

```diff
@@ -0,0 +1,17 @@
+import { useEffect } from 'preact/hooks'
+
+export const useGlobalEndCallKey = (callback) => {
+  useEffect(() => {
+    const onKeyEndCall = (e) => {
+      const key = e.key.toString()
+
+      if (key === 'EndCall') {
+        callback()
+        e.stopPropagation()
+        e.preventDefault()
+      }
+    }
+
+    document.addEventListener('keydown', onKeyEndCall)
```

Comment:
This function needs to return a unregister function

Reply:
I'm wonder if the unregister function is needed, because this is the "global" event that not going to remove in any places, 

and I'm thinking is that better to have a guard to make sure this define only once.